### PR TITLE
feat: add action confirm and adjust alert

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Description and Example

Add action Confirm.
Adjust Alert was missing one parameter

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.
- [X] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

**Review by matheusribeirozup — DISMISSED** [review]

File: beagle/lib/src/action/beagle_alert.dart [hunk 1]
```diff
@@ -21,11 +21,16 @@ class BeagleAlert {
     BuildContext context, {
     String title,
     String message,
+    String labelOk,
     Function onPressOk,
   }) {
     showDialog(
```
Comment (botbeaglezup) [inline beagle/lib/src/action/beagle_alert.dart:27]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/e9faf82f5120f133446357fc419bca5b5e45c75e/beagle/lib/src/action/beagle_alert.dart#L27"></span>The type argument(s) of the function 'showDialog' can't be inferred</td>
    </tr>
  </tbody>
</table>

File: beagle/lib/src/action/beagle_confirm.dart [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  static void showAlertDialog(
+    BuildContext context, {
+    String title,
+    String message,
+    String labelOk,
+    Function onPressOk,
+    String labelCancel,
+    Function onPressCancel,
+  }) {
+    showDialog(
```
Comment (botbeaglezup) [inline beagle/lib/src/action/beagle_confirm.dart:29]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/e9faf82f5120f133446357fc419bca5b5e45c75e/beagle/lib/src/action/beagle_confirm.dart#L29"></span>The type argument(s) of the function 'showDialog' can't be inferred</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-flutter/blob/e9faf82f5120f133446357fc419bca5b5e45c75e/beagle/test/bridge_impl/global_context_js_test.dart#L97">beagle/test/bridge_impl/global_context_js_test.dart#L97</a> - The type argument(s) of the function 'get' can't be inferred</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-flutter/blob/e9faf82f5120f133446357fc419bca5b5e45c75e/beagle/test/bridge_impl/global_context_js_test.dart#L109">beagle/test/bridge_impl/global_context_js_test.dart#L109</a> - The type argument(s) of the function 'get' can't be inferred</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-flutter/blob/e9faf82f5120f133446357fc419bca5b5e45c75e/beagle_components/lib/src/beagle_screen.dart#L55">beagle_components/lib/src/beagle_screen.dart#L55</a> - 'textTheme' is deprecated and shouldn't be used. This property is no longer used, please use toolbarTextStyle and titleTextStyle instead. This feature was deprecated after v2.4.0-0.0.pre.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_flutter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: beagle/lib/src/action/beagle_alert.dart [hunk 3]
```diff
@@ -17,15 +17,20 @@
...
+  static Future<void> showAlertDialog(
     BuildContext context, {
     String title,
     String message,
+    String labelOk,
     Function onPressOk,
-  }) {
-    showDialog(
+  }) async {
+    await showDialog(
```
Comment (botbeaglezup) [inline beagle/lib/src/action/beagle_alert.dart:27]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/b56aef5780ea6052ffaf664ecc6f9ee3f7a844e0/beagle/lib/src/action/beagle_alert.dart#L27"></span>The type argument(s) of the function 'showDialog' can't be inferred</td>
    </tr>
  </tbody>
</table>

File: beagle/lib/src/action/beagle_confirm.dart [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  static Future<void> showAlertDialog(
+    BuildContext context, {
+    String title,
+    String message,
+    String labelOk,
+    Function onPressOk,
+    String labelCancel,
+    Function onPressCancel,
+  }) async {
+    await showDialog(
```
Comment (botbeaglezup) [inline beagle/lib/src/action/beagle_confirm.dart:29]: <table data-meta="generated_by_flutter">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-flutter/blob/b56aef5780ea6052ffaf664ecc6f9ee3f7a844e0/beagle/lib/src/action/beagle_confirm.dart#L29"></span>The type argument(s) of the function 'showDialog' can't be inferred</td>
    </tr>
  </tbody>
</table>

**Review by hernandazevedozup — APPROVED** [review]

**Pull request merged**